reference/configuration/framework.rst

```diff
@@ -1560,6 +1561,16 @@ paths
 This option allows to define an array of paths where the component will look
 for translation files.
 
+.. _reference-translator-default_path:
+
+default_path
+.....
```

Comment:
The format we use for the docs is more strict that Markdown, so for example, the line below the heading must have the exact same length. Please, add the missing dots like this:

```rst
default_path
............
```